SpringProject/src/it/java/com/gradproject2019/user/UserControllerIT.java

```diff
@@ -141,6 +144,28 @@ public void shouldReturn401WhenUnauthorisedEdit() throws URISyntaxException {
         Assert.assertEquals("User unauthorized to perform action.", response.getBody().getMessage());
     }
 
+    @Test
+    public void shouldReturn200AndEmptyListWhenDatabaseEmpty() throws URISyntaxException {
```

Comment:
'WhenDatabaseEmpty' is not very BDD friendly - maybe something like `WhenNoUsers` would be better